src/flamegraph/mod.rs

```diff
@@ -103,6 +103,9 @@ where
     svg::write_header(&mut svg, imageheight)?;
     svg::write_prelude(&mut svg, imageheight, &opt)?;
 
+    // struct to reuse accross loops to avoid allocations
+    let event_start = &mut Event::Start({ BytesStart::owned_name("g") });
```

Comment:
```suggestion
    let mut event_start = Event::Start({ BytesStart::owned_name("g") });
```

Reply:
I'm surprised that that even works?

Reply:
Yes `cargo test` runs. what is surprising ?